tests/unit/flow/test_flow_multimode.py

```diff
@@ -45,6 +46,13 @@ def encode(self, data: str, *args, **kwargs) -> 'np.ndarray':
         return np.array(output)
 
 
+@pytest.fixture(autouse=True)
+def run_around_tests():
+    yield
+    rm_files(['vec1.gz', 'vec2.gz', 'chunk1.gz', 'chunk2.gz',
+              'vecidx1.bin', 'vecidx2.bin', 'kvidx1.bin', 'kvidx2.bin'])
+
+
```

Comment:
We should get rid of the `rm_files` method, but rather you pytest temporary folders. The `rm_files` wiped my whole home folder, since I used it a bit wrongly... Don't even know, what I did wrong, since it was wiped... :D